### PR TITLE
Add ignore option to scan task

### PR DESCRIPTION
**Pull request by cc1234475**
Check if the directory had a .ignore file, if so skip it.

### CODE REVIEW

File: internal/manager/task_scan.go [hunk 1]
```diff
@@ -378,6 +378,11 @@ func walkFilesToScan(s *models.StashConfig, f filepath.WalkFunc) error {
 				return filepath.SkipDir
 			}
 
+			ignore_exists, _ := fsutil.FileExists(filepath.Join(path, ".ignore"))
+			if ignore_exists {
+				return filepath.SkipDir
```
Comment (kermieisinthehouse) [inline internal/manager/task_scan.go:383]: Perhaps we should settle on a .stashignore name or something, if this isn't going to be configurable.

**Review by kermieisinthehouse — APPROVED** [review]

**Comment by WithoutPants** [comment]
I think this should be configurable. Under `Exclusions` settings, something like "Exclude directories where this filename is found".

**Comment by kermieisinthehouse** [comment]
I personally think it's fine - Plex doesn't make `.plexignore` configurable, for example. I'm not sure that we have the span yet that `.stashignore` shows up in archives and such - the only ones on your system will be ones you create

**Comment by WithoutPants** [comment]
I think it's more about avoiding redundancy. You might already have `.ignore` or `.plexignore` files in your directory, and it would be nice to have it play nice with those instead of having to rename or add a bunch of new files.

**Comment by kermieisinthehouse** [comment]
I didn't think of that usecase. Agreed.

**Comment by WithoutPants** [comment]
Closing this for inactivity. This will need to be redone for the new scanning engine anyway, and it needs to be configurable.

**Pull request closed without merging**